test/Spectre.Console.Cli.Tests/Unit/CommandAppTests.Help.cs

```diff
@@ -250,6 +252,62 @@ public Task Should_Output_Default_Command_And_Additional_Commands_When_Default_C
             return Verifier.Verify(result.Output);
         }
 
+        // Localised version of the above test, in English
+        [Theory]
+        [InlineData(null)]
+        [InlineData("")]
+        [InlineData("en")]
+        [InlineData("en-EN")]
+        [Expectation("Default_Without_Args_Additional")]
+        public Task Should_Output_Default_Command_And_Additional_Commands_When_Default_Command_Has_Required_Parameters_And_Is_Called_Without_Args_EN(string culture)
+        {
+            // Given
+            var fixture = new CommandAppTester();
+            fixture.SetDefaultCommand<LionCommand>();
+            fixture.Configure(configurator =>
+            {
+                configurator.AddExample("20", "--alive");
+                configurator.SetApplicationCulture(culture);
+                configurator.SetApplicationName("myapp");
+                configurator.AddCommand<GiraffeCommand>("giraffe");
+            });
+
+            // When
+            var result = fixture.Run();
+
+            // Then
+            var settings = new VerifySettings();
+            settings.DisableRequireUniquePrefix();
+            return Verifier.Verify(result.Output, settings);
+        }
+
+        // Localised version of the above test, in French
+        [Theory]
+        [InlineData("fr")]
+        [InlineData("fr-FR")]
+        [Expectation("Default_Without_Args_Additional-FR")]
+        public Task Should_Output_Default_Command_And_Additional_Commands_When_Default_Command_Has_Required_Parameters_And_Is_Called_Without_Args_FR(string culture)
+        {
+            // Given
+            var fixture = new CommandAppTester();
+            fixture.SetDefaultCommand<LionCommand>();
+            fixture.Configure(configurator =>
+            {
+                configurator.AddExample("20", "--alive");
+                configurator.SetApplicationCulture(culture);
+                configurator.SetApplicationName("myapp");
+                configurator.AddCommand<GiraffeCommand>("giraffe");
+            });
+
+            // When
+            var result = fixture.Run();
+
+            // Then
+            var settings = new VerifySettings();
+            settings.DisableRequireUniquePrefix();
+            return Verifier.Verify(result.Output, settings);
```

Comment:
I think it should be possible to re-use the `EN` method like this:

```suggestion
           return  Should_Output_Default_Command_And_Additional_Commands_When_Default_Command_Has_Required_Parameters_And_Is_Called_Without_Args_EN(culture);
```

That would spare me copying the content for a third time when I add the German translation.

Reply:
You mean collapsing both of these tests together @nils-a ?

```csharp
        // Localised version of the above test, in English
        [Theory]
        [InlineData(null)]
        [InlineData("")]
        [InlineData("en")]
        [InlineData("en-EN")]
        [Expectation("Default_Without_Args_Additional")]
        public Task Should_Output_Default_Command_And_Additional_Commands_When_Default_Command_Has_Required_Parameters_And_Is_Called_Without_Args_EN(string culture)
```

and

```csharp
        // Localised version of the above test, in French
        [Theory]
        [InlineData("fr")]
        [InlineData("fr-FR")]
        [Expectation("Default_Without_Args_Additional-FR")]
        public Task Should_Output_Default_Command_And_Additional_Commands_When_Default_Command_Has_Required_Parameters_And_Is_Called_Without_Args_FR(string culture)
```

I think an elegant solution might be to dynamically vary the expectation by appending the culture name to something like "Default_Without_Args_Additional-"

However, the following cases:

        [InlineData(null)]
        [InlineData("")]

Might actually need to become something like:

        [InlineData(null, "en")]
        [InlineData("", "en")]

In order to qualify the localisation result we expect ie. qualifying the expectation.